ibis/common/annotations.py

```diff
@@ -108,9 +127,9 @@ class Argument(Annotation):
 
     def __init__(
         self,
-        validator: Validator = None,
-        default: Any = EMPTY,
-        typehint: type = None,
+        validator: Optional[Validator] = None,
```

Comment:
Can't we use `Validator | None`?

Reply:
No we cannot because we cannot evaluate the `"Validator | None"` string (due to using future annotations) on python 3.8